src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1880,6 +1891,8 @@ else if (con.getTrustManagerClass() != null) {
                     && (SQLServerException.getErrString("R_truncatedServerResponse").equals(errMsg)
                             || SQLServerException.getErrString("R_truncatedServerResponse").equals(causeErrMsg))) {
                 con.terminate(SQLServerException.DRIVER_ERROR_INTERMITTENT_TLS_FAILED, form.format(msgArgs), e);
+            } else if (e instanceof FileNotFoundException) {
```

Comment:
This exception to be handled from the method that reads from the file, not here.

Reply:
ok, I've reverted it to the way it was being handled before.